src/fetchers/externalYieldRateFetcher.ts

```diff
@@ -93,22 +94,27 @@ export const fetchSushiForkYield = async (poolAddress: string, chainId?: number)
   return yieldRate
 }
 
-export async function fetchBenqiYield(underlyingAddress: string): Promise<number> {
-  const networkInfo: NetworkInfo = distributeConstantsByNetwork(43114);
-  const benqiAPI = 'https://api.benqi.fi/tokens/total_apys';
-  const response = await axios.get(benqiAPI).then((res: any) => res.data);
-  for (const t in networkInfo.contractAddresses.tokens) {
-    if (isSameAddress(networkInfo.contractAddresses.tokens[t], underlyingAddress)) {
-      if (t == "WETH") return response["AVAX"].supply;
-      return response[t].supply;
-    }
+export async function fetchBenqiYield(underlyingAddress: string, provider?: providers.JsonRpcProvider, chainId?: number): Promise<number> {
```

Comment:
let's change the naming from `underlyingAddress` to `qiTokenAddress` for better clarity

Reply:
done